publication/ver11/societal_impact.md

```diff
@@ -0,0 +1,239 @@
+
+### 7. [What are the power dynamics at play in implementations of your specification?](https://w3ctag.github.io/societal-impact-questionnaire/#power-dynamics)
+The WoT standard can be used to make details of how to access an IoT
+device available to a wider audience.  In particular, the manufacturer does
+not have to provide the metadata, it can be provided by a third party.  
+While this is not ideal ("official" support from a manufacturer would be more
+trustworth, third-party information would have to be used with care), it does
+mean that a manufacture will not be able to control access by restricting
+access to WoT Thing Descriptions.  They can simply be sourced from elsewhere,
+with some security caveats.
+
+The WoT also makes it easier to develop "local" IoT systems that do not
+require the cloud to function and can better manage private data.  They can
+in particular allow data from devices from different manufacturers to be used
+as part of a single system without that data having to go through a cloud
+API or service presented by that manufacturer.
+
+Finally, standardizing WoT metadata allows IoT devices from different manufacturers
+to be interchanged, reducing customer "lockin" to proprietary technologies or
+devices from a single manufacturer.
+
+In general, WoT gives users and the implementers of IoT systems more information
+about IoT devices and more control over how they are deployed.
+
+### 8. [What points of centralization does your feature bring to the web platform?](https://w3ctag.github.io/societal-impact-questionnaire/#centralization)
+This iteration of the WoT Architecture introduces WoT Discovery, which includes
+a specification for a WoT Directory service.  This provides a searchable
+database of WoT Thing Descriptions.  However, there can be many such services
+used in different contexts; they do not need to be centralized, although such 
+centralization may be useful in some contexts (e.g. a central directory for
+smart city services in a particular city).
+WoT Discovery is also an optional service and is not required for 
+deployment of a WoT system.
+
+Identifiers used in WoT Thing Descriptions can be generated randomly using
+UUIDs, avoiding the need for a central id management service.  This is recommended
+but not required, so some deployments of WoT may choose to use a centralized
+identifier allocation scheme.
+
+Ideally, WoT Thing Models should have a fixed URL and be available from a single
+source.  It would, however, be possible to cache and duplicate Thing Models, which
+would also avoid the privacy hazard of looking up Thing Models dynamically.
+
+### 9. [To what extent do the features in your specification impact the natural environment?](https://w3ctag.github.io/societal-impact-questionnaire/#environment)
+IoT systems generally can impact the environment in both positive and negative
+ways.  Negatively, use of IoT implies a proliferation of small electronic devices
```

Comment:
I would mention that a connected device may have less durability than the device it aims to replace, e.g an electromechanical switch vs a connected actuator. But this is mainly a market-specific, hopefully transient, situation.